CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 Change log
 -----------
 
+* Add versionist support [Florin]
```

Comment:
This will be automatically generated by versionist as `versionist.conf.js: Add versionist config file [Florin]`

Reply:
Yeah, but I think this will happen only after versionist is merged

Reply:
Right? ^

Reply:
Yeah, but I would say to wait for that before merging this, I just want to be sure that the various `versionist.conf.js` files don't have to be changed several times